vnext/PropertySheets/React.Cpp.props

```diff
@@ -21,6 +21,7 @@
   <PropertyGroup Label="Desktop">
     <!-- See https://docs.microsoft.com/en-us/cpp/porting/modifying-winver-and-win32-winnt -->
     <WinVer>_WIN32_WINNT_WIN8</WinVer>
+    <EnableBeast Condition="'$(EnableBeast)' == ''">0</EnableBeast>
```

Comment:
Nit: idiom is to test against the string "true" instead of using numbers

Reply:
I considered it, but the value would need to be translated again into `1` or `0` to use in the C++ macro `ENABLE_BEAST` (via additional `ItemDefinitionGroup` entry).
For a transitory property, I think it should be fine to use an unorthodox convention.